src/google/protobuf/io/coded_stream.h

```diff
@@ -112,7 +112,7 @@
 #include <string>
 #include <utility>
 #ifdef _MSC_VER
-  #if defined(_M_IX86) && \
+  #if (defined(_M_IX86) || defined(_M_AMD64)) && \
```

Comment:
Please update this to be the same as in port.h:
https://github.com/google/protobuf/blob/master/src/google/protobuf/stubs/port.h#L49

From what I gathered, windows always uses little-endian:
http://stackoverflow.com/questions/6449468/can-i-safely-assume-that-windows-installations-will-always-be-little-endian